dns_test.go

```diff
@@ -567,9 +568,10 @@ func TestUpdateDNSRecord_ClearComment(t *testing.T) {
 	setup()
 	defer teardown()
 
+	emptyComment := ""
 	input := DNSRecord{
 		ID:      "372e67954025e0ba6aaa6d586b9e0b59",
-		Comment: "",
+		Comment: &emptyComment,
```

Comment:
similar to https://github.com/cloudflare/cloudflare-go/pull/1393/files#r1320954470